rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/ParticipantCreator.java

```diff
@@ -33,10 +37,17 @@ Participant from(ParticipantDTO participantDTO) {
     final HealthReport healthReport = fromValue(participantDTO.getHealthReport());
     participantBuilder.healthReport(healthReport);
 
-    final Experience experience = fromValue(participantDTO.getExperience());
+    final ExperienceValue experienceValue = participantDTO.getExperience();
+    final Experience experience = fromValue(experienceValue);
```

Comment:
Operacje w dwóch linijkach robisz dla czytelności?
Dla HealthReport jest wykonana w jednej linijce

Reply:
to nie jest dla czytelności. Zobacz niżej, w linii 46 wykorzystuję to `experienceValue`. Cała ta operacja i skomplikowanie końcówki tej metody jest po to, żeby podwiązać te całe historicalTurns do konkretnego uczestnika, żeby była dwustrona relacja one-to-many. Dużo trzeba kombinować niestety, ale nie mam pomysłu na razie jak to uprościć.